src/services/formatting/rules.ts

```diff
@@ -213,25 +213,15 @@ namespace ts.formatting {
         public NoSpaceBetweenYieldKeywordAndStar: Rule;
         public SpaceBetweenYieldOrYieldStarAndOperand: Rule;
 
-        // Async-await
+        // Async functions
         public SpaceBetweenAsyncAndFunctionKeyword: Rule;
-        public NoSpaceBetweenAsyncAndFunctionKeyword: Rule;
-        public SpaceAfterAwaitKeyword: Rule;
-        public NoSpaceAfterAwaitKeyword: Rule;

```

Comment:
The other `NoSpace` ones should not exist. I once thought it will work together with `Space` rules but then found they should only exist when there are some valid formatting options.